client/mobile/components/teacher/requestFeedbackView.js

```diff
@@ -66,33 +68,27 @@ class RequestFeedbackView extends React.Component {
 
   renderFeedbackOptions(feedbackOptions) {
     return feedbackOptions.map((feedbackOption, index) => {
-      return (
-        <View style={styles.buttonContainer} key={index}>
-          <TouchableOpacity onPress={this.selectFeebackOption.bind(this, feedbackOption)} style={styles.button}>
-            <Text style={styles.buttonText}> {feedbackOption.name} </Text>
-          </TouchableOpacity>
-        </View>
+      return (      
+        <Button key={index} onPress={this.selectFeebackOption.bind(this, feedbackOption)} style={styles.button} text={feedbackOption.name} />
```

Comment:
isn't style set in the button? What does this style prop do? What if we want different color buttons on different pages?